hs_file_types/models/reftimeseries.py

```diff
@@ -495,26 +495,26 @@ def get_html(self):
 
         html_string = super(RefTimeseriesFileMetaData, self).get_html()
         if not self.has_metadata:
-            root_div = div(cls="alert alert-warning alert-dismissible", role="alert")
-            with root_div:
-                h4("No file level metadata exists for the selected file.")
-            html_string = root_div.render()
+            html_string = '<div id="#fb-metadata-default" class="text-center text-muted" ' \
```

Comment:
Again, why move away from dominate to a string?

Reply:
In fact this error message is not needed as ref time series will always have metadata. Removing the string and check for metadata.